src/puppetlabs/trapperkeeper/services/status/status_core.clj

```diff
@@ -95,6 +95,40 @@
                  version))))
     version))
 
+(defn level->int
+  "Returns an integer which represents the given status level.
+   The ordering of levels is :critical < :info < :debug."
+  [level]
+  (case level
+    :critical 0
+    :info 1
+    :debug 2))
+
+(defn apply-to-int-levels
```

Comment:
That said, I don't like this name very much and would be happy to change it to something else - I just think it needs to accurately reflect the behavior.